projects/common/src/navigation/navigation.service.ts

```diff
@@ -73,6 +73,22 @@ export class NavigationService {
     return this.currentParamMap.getAll(parameterName);
   }
 
+  public constructExternalUrl(urlString: string): string {
```

Comment:
thanks! looks good - with these changes, do we still need the changes in time range service or the new url util?

Reply:
nope, I already reverted those changes.

Reply:
oh right, the TR ones are gone - one of the url utils looks like it's still there being unused though